dictdiffer/__init__.py

```diff
@@ -29,15 +27,11 @@
 SET_TYPES = (MutableSet, )
 
 try:
-    pkg_resources.get_distribution('numpy')
-except pkg_resources.DistributionNotFound:  # pragma: no cover
-    HAS_NUMPY = False
-else:
-    HAS_NUMPY = True
-
     import numpy
-
+    HAS_NUMPY = True
     LIST_TYPES += (numpy.ndarray, )
+except ImportError:  # pragma: no cover
```

Comment:
Using `ImportError` is not a good practice. Do you have any issue with using `pkg_resources`?

Reply:
I did not know about `ImportError`being a bad practice. Can you share any links related to this?
We do not ship setuptools so at the moment dictdiffer can only be used in a dev environment on our side.

Reply:
As an alternative I would use [`importlib.find_spec`](https://docs.python.org/3/library/importlib.html#importlib.machinery.PathFinder.find_spec) but we would have to remove support for Python 2 which might be still a bit too early for some users.

Reply:
> Can you share any links related to this?

It used to be in quantifiedcode.com but the project is dead now. Hence, I don't have anything to support my argument anymore.